Modules/_asynciomodule.c

```diff
@@ -1602,37 +1603,55 @@ FutureIter_dealloc(futureiterobject *it)
     }
 }
 
-static PyObject *
-FutureIter_iternext(futureiterobject *it)
+static PySendResult
+FutureIter_am_send(futureiterobject *it, PyObject *arg, PyObject **result)
```

Comment:
Is `arg` ignored?

Reply:
yes, there is a comment in `FutureIter_send`, I  can copy it here as well:
```
    /* Future.__iter__ doesn't care about values that are pushed to the
     * generator, it just returns self.result().
     */
```